### PR TITLE
gh-109414: Add some basic information about venvs in the introduction.

### PR DESCRIPTION
**Pull request by vsajip**
* Issue: gh-109414

----
:books: Documentation preview :books:: https://cpython-previews--109440.org.readthedocs.build/en/109440/library/venv.html

### CODE REVIEW

File: Doc/library/venv.rst [hunk 1]
```diff
@@ -30,6 +30,23 @@ When used from within a virtual environment, common installation tools such as
...
+A virtual environment is:
+
+* Used to contain a specific Python interpreter and software libraries and
+  binaries which are needed to support a project (library or application).
+
+* Contained in a directory, conventionally either named ``venv`` or ``.venv`` in
+  the project directory, or under a container directory for lots of virtual
+  environments, such as ``~/.virtualenvs``.
+
+* Not checked into source control systems such as Git, Mercurial etc.
```
Comment (vstinner) [inline Doc/library/venv.rst]: I don't think that you need to give multiple examples:

```suggestion
* Not checked into source control systems such as Git.
```

File: Doc/library/venv.rst [hunk 2]
```diff
@@ -30,6 +30,23 @@ When used from within a virtual environment, common installation tools such as
...
+
+* Considered as disposable -- it should be simple to delete and recreate it from
+  scratch. You don't place any project code in the environment
+
+* Not considered as movable or copyable -- you just create a new environment in
```
Comment (vstinner) [inline Doc/library/venv.rst]:
```suggestion
* Not considered as movable or copyable -- you just recreate the same environment in
```

**Comment by vstinner** [comment]
I'm using venv as a sandbox to install libraries that I don't want to install system-wide, but your definition implies that.

I'm also using them to modify the code: it's convenient to develop, since changes are "undone" when I remove (and recreate) the venv.

**Review by vstinner — APPROVED** [review]
LGTM. I made some suggestions, but you can ignore them ;-)

**Comment by vsajip** [comment]
> I'm using venv as a sandbox to install libraries that I don't want to install system-wide, but your definition implies that.
> 
> I'm also using them to modify the code

Yes, the list isn't meant to be exhaustive :wink:

**Review by vstinner — APPROVED** [review]
LGTM, thanks for the minor updates 😉

**Review by brettcannon — APPROVED** [review]

**Pull request merged**